configurations/fig-validation-service.conf

```diff
@@ -1,6 +1,12 @@
-TEMPLATE_DIRECTORY='openshift'
-SOURCE_CONTEXT_DIR='/src/figvalidationserivce'
-BUILD_CONFIG_TEMPLATE='fig-validation-service.bc.yaml'
-DEPLOY_CONFIG_TEMPLATE='fig-validation-service.dc.yaml'
-APP_NAME='fig-validation-service'
-BUILD_REQUIRED='false'
+###############################################################################
+##
+##            DPS Validation Service Specific Environment Variables
+##            Author: @akroon3r
+##
+###############################################################################
+
+APP_NAME="fig-validation-service"
+BUILD_CONFIG_TEMPLATE="fig-validation-service.bc.yaml"
+BUILD_REQUIRED="false"
+DEPLOY_CONFIG_TEMPLATE="fig-validation-service.dc.yaml"
+SOURCE_CONTEXT_DIR="/src/figvalidationserivce"
```

Comment:
the fig validation service should use the following env to connect to ORDS:

- FIGCR_BASE_URL
- FIGCR_USERNAME
- FIGCR_PASSWORD

Reply:
Thanks, we'll need to pull that into the containers.  I'll add this to the PR where I pull out the packing material referenced below.

Reply:
added blank values to the config